.github/workflows/ansible-ci.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: CI - Linting
+
+'on':
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
These lines diverge from the otherwise pretty consistently applied style of using double quotes for YAML and avoiding `[listelement]` notation for lists. Not sure if intentional.

(compare e.g. with `.github/workflows/galaxy-publish.yml`)